### PR TITLE
feat: new progress bar component added to dashboard

### PR DESCRIPTION
**Pull request by nehasacher143**
**Task:** https://bloomstack.com/desk#Form/Task/TASK-2021-00243

**Screenshot:**
screenshot-localhost_8005-2021 09 20-11_14_15

### CODE REVIEW

File: frappe/public/js/frappe/form/dashboard.js [hunk 1]
```diff
@@ -90,6 +90,24 @@ frappe.ui.form.Dashboard = Class.extend({
 		return progress_chart;
 	},
 
+	add_progress_chart: function(title, percent) {
+		var progress_chart = this.make_progress_chart(title);
```
Comment (hrwX) [inline frappe/public/js/frappe/form/dashboard.js]:
```suggestion
		let progress_chart = this.make_progress_chart(title);
```

File: frappe/public/js/frappe/form/dashboard.js [hunk 2]
```diff
@@ -90,6 +90,24 @@ frappe.ui.form.Dashboard = Class.extend({
...
+
+		if(!$.isArray(percent)) {
```
Comment (hrwX) [inline frappe/public/js/frappe/form/dashboard.js]:
```suggestion
		if (!$.isArray(percent)) {
```

File: frappe/public/js/frappe/form/dashboard.js [hunk 3]
```diff
@@ -90,6 +90,24 @@ frappe.ui.form.Dashboard = Class.extend({
...
+			percent = this.format_percent(title, percent);
+		}
+
+		var progress = $('<ul class="progress_chart"></ul>').appendTo(progress_chart);
```
Comment (hrwX) [inline frappe/public/js/frappe/form/dashboard.js]:
```suggestion
		let progress = $('<ul class="progress_chart"></ul>').appendTo(progress_chart);
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by nehasacher143** [comment]
@sahil28297 @MyuddinKhatri @neilLasrado @DeveloperMetal  Please review this one

**Review by neilLasrado — APPROVED** [review]

**Review by DeveloperMetal — APPROVED** [review]

**Pull request closed without merging**